config-templates/ibm/ibm-vpc-block-csi-driver/4.3.0/storage-class-parameters.json

```diff
@@ -0,0 +1,143 @@
+[
+    {
+        "description": "Enter a name for your storage class.",
+        "displayname": "Storage class name",
+        "name": "name",
+        "templateKeyName": [
+            "name"
+        ],
+        "required": "true",
+        "category": "config"
+    },
+    {
+        "description": "Enter the reclaim policy for your storage class. If you want to keep the PV, the physical storage device and your data when you remove the PVC, enter Retain. If you want to delete the PV, the physical storage device and your data when you remove the PVC, enter Delete.",
+        "displayname": "Reclaim policy",
+        "name": "reclaimPolicy",
+        "templateKeyName": [
+            "reclaimPolicy"
+        ],
+        "default": "Retain",
+        "required": "false",
+        "category": "config"
+    },
+    {
+        "description": "Choose if you want to delay the creation of the Block Storage for VPC instance until the first pod that uses this storage is ready to be scheduled. To delay the creation, enter 'WaitForFirstConsumer'. To create the instance when you create the PVC, enter 'Immediate'.",
+        "displayname": "Volume binding mode",
+        "name": "volumeBindingMode",
+        "templateKeyName": [
+            "volumeBindingMode"
+        ],
+        "default": "WaitForFirstConsumer",
+        "required": "false",
+        "category": "config"
+    },
+    {
+        "description": "Enter pre-defined profile (10iops-tier, 5iops-tier, 3iops-tier), or enter 'custom' to use a custom IOPs value. To find supported storage sizes for a specific profile, see https://cloud.ibm.com/docs/vpc?topic=vpc-block-storage-profiles. Any PVC that uses this storage class must specify a size value that is within this range.",
```

Comment:
@prankulmahajan Please check last sentence of description. I hope this does not confuse user
`Any PVC that uses this storage class must specify a size value that is within this range`

Reply:
@mssachan I will make it more clear in upcoming PR